src/azure-cli/azure/cli/command_modules/iot/_params.py

```diff
@@ -78,55 +78,22 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                    'cross geo-pair disaster recovery. This property is immutable once set on the resource. '
                    'Only available in select regions. Learn more at https://aka.ms/dpsdr')
 
-    # To deprecate
-    for subgroup in ['access-policy', 'linked-hub', 'certificate']:
+    for subgroup in ['policy', 'linked-hub', 'certificate']:
         with self.argument_context('iot dps {}'.format(subgroup)) as c:
             c.argument('dps_name', options_list=['--dps-name'], id_part=None)
 
-    # To replace deprecated
-    for subgroup in ['policy']:
-        with self.argument_context('iot dps {}'.format(subgroup)) as c:
-            c.argument('dps_name', options_list=['--dps-name', '-n'], id_part=None)
-
-    with self.argument_context('iot dps access-policy') as c:
-        c.argument('access_policy_name', options_list=['--access-policy-name', '--name', '-n'],
-                   help='A friendly name for DPS shared access policy.')
-
-    with self.argument_context('iot dps access-policy create') as c:
-        c.argument('rights', options_list=['--rights', '-r'], nargs='+',
-                   arg_type=get_enum_type(AccessRightsDescription),
-                   help='Access rights for the IoT Hub Device Provisioning Service. '
-                        'Use space-separated list for multiple rights.')
-        c.argument('primary_key', help='Primary SAS key value.')
-        c.argument('secondary_key', help='Secondary SAS key value.')
-
-    with self.argument_context('iot dps access-policy update') as c:
-        c.argument('rights', options_list=['--rights', '-r'], nargs='+',
-                   arg_type=get_enum_type(AccessRightsDescription),
-                   help='Access rights for the IoT Hub Device Provisioning Service. '
-                        'Use space-separated list for multiple rights.')
-        c.argument('primary_key', help='Primary SAS key value.')
-        c.argument('secondary_key', help='Secondary SAS key value.')
-
     with self.argument_context('iot dps policy') as c:
-        c.argument('access_policy_name', options_list=['--policy-name', '--pn'],
+        c.argument('access_policy_name', options_list=['--policy-name', '--pn', '--name', '-n'],
```

Comment:
Why add --name and -n as arg aliases for policy name? We used this deprecation opportunity to explicitly align arg aliases with the IoT extension command set where `-n` generally refers to the target parent ARM resource in question, which is a pivot from the old access-policy.

This is the presentation with 2.34.1 which is what we wanted to align on.
![image](https://user-images.githubusercontent.com/28658112/161659973-12f2f74e-297c-40ad-95cb-78e2c84bb1d3.png)

Reply:
done - now dps_name refers to --dps-name, --name, and -n

![image](https://user-images.githubusercontent.com/73560279/161840818-ac68b2e1-ac59-437a-b34b-0a761cf285be.png)

Reply:
Thank you.

I believe we will need new test recordings for the changed tests.